### PR TITLE
acronym sweep

### PR DESCRIPTION
**Pull request by HettieS**
### Trello card
https://trello.com/c/KcMeOzds/1510-acronym-sweep-ensure-acronyms-are-spelled-out-on-each-page-theyre-referenced
### Context
Noticed we are missing some acronym tooltips, so adding in the most frequently found terms. 
### Changes proposed in this pull request
Adding a few new acronyms into `frontmatter.yml`
### Guidance to review
- check the acronyms added are correct in full
- check for typos
- spot check a couple of newly added acronyms on site to make sure tooltip working as expected

### CODE REVIEW

**Review by MylesJarvis — APPROVED** [review]
LGTM

File: config/frontmatter.yml [hunk 1]
```diff
@@ -19,3 +19,16 @@ acronyms:
...
   EYITT: Early Years Initial Teacher Training
+  SEND: Special Education Needs and Disabilities
+  FE: Further Education
+  AO: Assessment Only
+  STEM: Science Technology Engineering Maths
+  EY: Early Years
+  TSA: Teaching School Alliance
+  EYTS: Early Years Teacher Status
+  CPD: Continuing Professional Development
+  EYITT: Early Years Initial Teacher Training
```
Comment (peteryates) [inline config/frontmatter.yml:30]: `EYTS` and `EYITT` are already present

**Review by peteryates — COMMENTED** [review]
Looks fine overall, just a couple of duplicates to remove 👍🏽

**Review by peteryates — CHANGES_REQUESTED** [review]
Just a couple of duplicates to remove ☝🏽

**Review by MylesJarvis — APPROVED** [review]

**Review by peteryates — APPROVED** [review]

**Pull request merged**